test/e2e/utils/pod.go

```diff
@@ -141,27 +142,20 @@ func NFDMasterPod(image string, onMasterNode bool) *corev1.Pod {
 			RestartPolicy:      corev1.RestartPolicyNever,
 		},
 	}
-	if onMasterNode {
```

Comment:
Was it removed on purpose?

Reply:
Yeah, it's replaced by `SpecWithMasterNodeSelector`. It's currently unused so it could be dropped, too 🤔